tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -371,14 +371,14 @@ export const getMetricsStepSelectorEnabled = createSelector(
 export const getMetricsStepSelectorRangeEnabled = createSelector(
```

Comment:
I think we should combine getMetricsStepSelectorRangeEnabled and getMetricsLinkedTimeRangeEnabled into getMetricsRangeEnabled. Why do we want two selectors which return the same value?

Reply:
I definitely should have done that initially, done now.